### PR TITLE
Add docs for Gauge instrument

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs [hunk 1]
```diff
@@ -75,13 +75,13 @@ public override ExportResult Export(in Batch<MetricItem> batch)
 
                         case MetricType.LongGauge:
                             {
-                                // TODOs
+                                valueDisplay = (metric as IGaugeMetric).LastValue.Value.ToString();
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs:78]: there are changes that are upcoming to this section. For now, this works (to demo the usage of Gauge), and the changes to this section are coming as separate PRs.

File: docs/metrics/getting-started-gauge/Program.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+public class Program
+{
+    private static readonly Meter MyMeter = new Meter("TestMeter", "0.0.1");
+    private static readonly Random RandomGenerator = new Random();
+
+    public static async Task Main(string[] args)
+    {
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddSource("TestMeter")
```
Comment (reyang) [inline docs/metrics/getting-started-gauge/Program.cs:33]: curious/nit - do we want to indent by 8 spaces or 4?

File: OpenTelemetry.sln [hunk 3]
```diff
@@ -214,7 +214,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentati
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule.Tests", "test\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule.Tests\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule.Tests.csproj", "{4D7201BC-7124-4401-AD65-FAB58A053D45}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "getting-started-histogram", "docs\metrics\getting-started-histogram\getting-started-histogram.csproj", "{92ED77A6-37B4-447D-B4C4-15DB005A589C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started-histogram", "docs\metrics\getting-started-histogram\getting-started-histogram.csproj", "{92ED77A6-37B4-447D-B4C4-15DB005A589C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "getting-started-gauge", "docs\metrics\getting-started-gauge\getting-started-gauge.csproj", "{E7F491CC-C37E-4A56-9CA7-8F77F59E0614}"
```
Comment (reyang) [inline OpenTelemetry.sln:219]: minor: consider calling it `getting-started-asynchronous-gauge` or `getting-started-async-gauge`.

**Review by reyang — APPROVED** [review]

File: docs/metrics/getting-started-gauge/Program.cs (hunk 2 above)
Comment (cijothomas) [reply docs/metrics/getting-started-gauge/Program.cs:33]: dont have preference. But it should be consistent! (agree this file itself is not consistent. will address it as part of a general docs cleanup effort)

**Pull request merged**